### ISSUE TITLE
`min` and `max` values in output lexicons

### ISSUE THREAD

**Issue by michaelcapizzi** [williamleif/socialsent#3]
First of all, thank you @williamleif for releasing this code.  It will be tremendously helpful in my research.

This is not really an `issue` as much as a question.

What's the `mininum` and `maximum` values of output lexicons?  It appears that it's somewhere around `+/- 4 or 5` but I can't find it identified anywhere in the original paper.

If it's in the code base somewhere I haven't found it either.

Thanks in advance for the quick help.

**Comment by williamleif** [comment]
Sorry for the slow response! This is a good question. In general, across the various methods implemented, there actually is no fixed min or max value. Some of the methods, such as the random_walk method and the main SentProp method described in the paper, do have min=0 and max=1 since they are defined based on random walk probabilities, but methods like densifier and pmi have no fixed range. 

However, for the lexicons I released, I standardized the values to have 0 mean and unit variance within each corpus (i.e., I z-scored them). This again means that the values in the lexicons I released have no fixed min or max (since standardizing in this way doesn't produce a fixed range), but the scores for each word can be interpreted as "how many positive/negative standard deviations is this word away from the sentiment of the average word in a corpus." As you noticed, the min and max after standardizing is usually around [-4,4], so the most sentiment-bearing words are around 4 standard deviations away from the average word. 

In general, the outputs from the various sentiment induction methods are somewhat "unit-less"; the ranges can be somewhat arbitrary, and it can be hard to interpret the absolute values of the scores. I recommend normalizing/standardizing within a particular dataset if you want to be able to interpret the value assigned to an individual word.
